tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1553,6 +1552,16 @@ def runTest(self):
         else:
             cell_occupancy = 1
 
+        pkt_dst_mac2 = router_mac if router_mac != '' else dst_port_2_mac
```

Comment:
Since it will use the router mac or vlan mac as the DMAC, why there are several different port mac, like dst_port_2_mac and dst_port_3_mac, when we expect these 2 mac be used in the test? No router_mac and vlan_mac? @XuChen-MSFT

Reply:
Destination of test traffic flow is PTF's port. So there are multiple dst port in test case.
and since this change is focus on dualtor device, regarding to single TOR environment, is going to collect more info from  various testbed test report. And then to know if it is necessary to have similar fix for single TOR. so add some logging about arp proxy config in this PR.